jax/numpy/lax_numpy.py

```diff
@@ -921,6 +921,19 @@ def histogram(a, bins=10, range=None, weights=None, density=None):
     counts = counts / bin_widths / counts.sum()
   return counts, bin_edges
 
+@_wraps(np.histogram2d)
+def histogram2d(x, y, bins=10, range=None, weights=None, density=None):
+
+  N = len(bins)
```

Comment:
It looks like this would result in a `TypeError` for the default value.

Reply:
Yes you're right